### DISCUSSION TITLE
Add Purolator and Intelcom shippers for Canada

### DISCUSSION THREAD

**Discussion by glbailey** [moralmunky/Home-Assistant-Mail-And-Packages#745] [Show and tell]
I've successfully added Purolator and Intelcom if anyone is interested. Intelcom is used by Amazon in our end of Canada so none of the amazon code is useful here but now Intelcom works well. Hope it helps someone.
[GB_added-shippers.txt](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/files/9962141/GB_added-shippers.txt)

**Comment by firstof9** [comment]
You can put this into a PR.

**Comment by glbailey** [comment]
Thankyou Chris. Did my best to figure out how to do a PR and hopefully all is well. Added 2 new shippers for Canada.
Garnet
